sale_blanket_order/wizard/create_sale_orders.xml

```diff
@@ -8,6 +8,11 @@
                 <group>
                     <field name="line_ids" nolabel="1">
                         <tree create="false" editable="bottom">
+                            <field
```

Comment:
Also this change: is it needed and why?

Reply:
Refers to my last comment: https://github.com/OCA/sale-workflow/pull/1785/files/6391c73a8f54d6682c6cf828e921bed7cd569d52..83b003ff3a3200a291c35ab04f36f434c08bee35#r773338386